couchpotato/core/notifications/emby.py

```diff
@@ -0,0 +1,89 @@
+import json
+import urllib, urllib2
+
+from couchpotato.core.helpers.variable import cleanHost
+from couchpotato.core.logger import CPLog
+from couchpotato.core.notifications.base import Notification
+
+
+log = CPLog(__name__)
+
+autoload = 'Emby'
+
+
+class Emby(Notification):
+
+    def notify(self, message = '', data = None, listener = None):
+        host = self.conf('host')
+        apikey = self.conf('apikey')
+
+        host = cleanHost(host)
+        url = '%semby/Library/Series/Updated' % (host)
```

Comment:
from speaking with @LukePulverenti he will add /Library/Movies/Updated with then next Emby release. So feel free to change my PR accordingly. Emby releases pretty often so I'm sure it will work at that URL soon.